content/_data/changelogs/weekly.yml

```
@@ -5088,7 +5088,7 @@
             url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#333
         message: |-
           Update Remoting from 3.29 to 3.33.
-    # pull: 4074 too minor (Use a proper example domain in hudson.FunctionsTest#testGetActionUrl_unparseable())
+    # pull: 4074 too minor (Use a proper example domain in hudson.FunctionsTest#testGetActionUrl_unparsable())
```

Correcting the method name seems unlikely to be accurate
```suggestion
    # pull: 4074 too minor (Use a proper example domain in hudson.FunctionsTest#testGetActionUrl_unparseable())
```

---

oops, testGetActionUrl_unparseable is correct, and I saw it, but missed it when I was adding exceptions